internal/cmd/base/profiling_on.go

```diff
@@ -4,11 +4,14 @@
 package base
 
 import (
+	"context"
 	"os"
 	"path/filepath"
 	"runtime"
 	"runtime/pprof"
 	"time"
+
+	"github.com/hashicorp/boundary/internal/observability/event"
```

Comment:
I ran `goimports` on the code base and it turned up this diff, I can move it to a separate PR if we want but I figured it was small enough.